### PR TITLE
Eliminating configuration Steps

### PR DESCRIPTION
**Pull request by ah1053**
To eliminate[ 2 steps in the PingPong tutorial ](https://developers.mobiledgex.com/guides-and-tutorials/how-to-workshop-adding-mobiledgex-matchingengine-sdk-to-unity-ping-pong-demo-app#for-ios)
The two steps to be eliminated are:
1. Adding PlatformIntegration.m to the generated XCode Project.
2. Adding CoreTelephony.framework to the Xcode Project. (Also Adding UIkit.framework to Unity 2019.3.5f)

Unity supports automated plug-in integration and copies all files with the following extensions to the generated Xcode project if you enable them for iOS in the Plugin Inspector
 window: .a, .m , .mm, .c, .cpp, .h. If any files with these extensions are located n the Assets/Plugins/iOS folder, Unity only enables them for the iOS platform.

### CODE REVIEW

File: unity/MexRestSample/Assets/Plugins/iOS/PlatformIntegration.m [hunk 1]
```diff
@@ -0,0 +1,247 @@
+/**
+ * Copyright 2019 MobiledgeX, Inc. All rights and licenses reserved.
```
Comment (brucearmstrong) [inline unity/MexRestSample/Assets/Plugins/iOS/PlatformIntegration.m:2]: Date 2020.

File: unity/PingPongGameExample/PingPongGame/Assets/Plugins/iOS/PlatformIntegration.m [hunk 2]
```diff
@@ -0,0 +1,247 @@
+/**
+ * Copyright 2019 MobiledgeX, Inc. All rights and licenses reserved.
```
Comment (brucearmstrong) [inline unity/PingPongGameExample/PingPongGame/Assets/Plugins/iOS/PlatformIntegration.m:2]: date

**Comment by franklin-huang-mobiledgex** [comment]
Can we remove the iosPongGame folder then?

File: unity/MexRestSample/Assets/Plugins/iOS/PlatformIntegration.m (hunk 1 above)
Comment (ah1053) [reply unity/MexRestSample/Assets/Plugins/iOS/PlatformIntegration.m:2]: Shall I change this date on all of the scripts on the MexRestSample and PingPong?

Comment (franklin-huang-mobiledgex) [reply unity/MexRestSample/Assets/Plugins/iOS/PlatformIntegration.m:2]: I think we are doing 2018-2020 @lgarner

Comment (franklin-huang-mobiledgex) [reply unity/MexRestSample/Assets/Plugins/iOS/PlatformIntegration.m:2]: If there are a lot of changes, that can be a separate PR

File: unity/MexRestSample/Assets/Editor/BuildPostProcessor.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            string unityFrameworkGUID = project.GetUnityFrameworkTargetGuid();
+
+            AddFrameworks(project, targetGUID, unityFrameworkGUID);
+
+            // Write.
+            File.WriteAllText(projectPath, project.WriteToString());
+        }
+    }
+
+    static void AddFrameworks(PBXProject project, string targetGUID,string unityFrameworkGUID)
```
Comment (lgarner) [inline unity/MexRestSample/Assets/Editor/BuildPostProcessor.cs:30]: Nit: Space.

File: unity/PingPongGameExample/PingPongGame/Assets/Editor/BuildPostProcessor.cs [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            string unityFrameworkGUID = project.GetUnityFrameworkTargetGuid();
+
+            AddFrameworks(project, targetGUID, unityFrameworkGUID);
+
+            // Write.
+            File.WriteAllText(projectPath, project.WriteToString());
+        }
+    }
+
+    static void AddFrameworks(PBXProject project, string targetGUID,string unityFrameworkGUID)
```
Comment (lgarner) [inline unity/PingPongGameExample/PingPongGame/Assets/Editor/BuildPostProcessor.cs:30]: Nit: space: ", string"

File: unity/PingPongGameExample/PingPongGameSkeleton/Assets/Editor/BuildPostProcessor.cs [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            string unityFrameworkGUID = project.GetUnityFrameworkTargetGuid();
+
+            AddFrameworks(project, targetGUID, unityFrameworkGUID);
+
+            // Write.
+            File.WriteAllText(projectPath, project.WriteToString());
+        }
+    }
+
+    static void AddFrameworks(PBXProject project, string targetGUID,string unityFrameworkGUID)
```
Comment (lgarner) [inline unity/PingPongGameExample/PingPongGameSkeleton/Assets/Editor/BuildPostProcessor.cs:30]: Same.

**Comment by brucearmstrong** [comment]
@ah1053, did you mean to close this without merging?

**Pull request closed without merging**